doc/source/install_linux_distro.rst

```diff
@@ -127,6 +127,7 @@ git repository and install it as a regular user::
 
   git clone https://gitlab.com/BuildStream/buildstream.git
   cd buildstream
+  git checkout <VERSION>
```

Comment:
Thanks @LawrenceHunter1 , unless this is replaced at build time with the latest stable tag, I think is better to change this to "git checkout bst-1"

Reply:
That works better for the latest version but it wouldn't resolve the same issue for example 1.4.2 which is also missing the branching command. This is based on my assumption when reading the docs you're reading the latest version, as the Installing from source page does not vary between which version is selected to install.

Reply:
I think you are right; but then you need to explain what "version" is (tag in the git repo), so there is no confusion